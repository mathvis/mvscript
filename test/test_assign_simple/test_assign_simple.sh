#!/bin/bash
helix test_assign_simple.mvs test_assign_simple.out --vsplit
