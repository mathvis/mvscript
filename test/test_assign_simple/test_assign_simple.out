Decl (Variable (VarIdentifier "a") (Just int) (Just (Type (Int 0))))
Decl (Assignment (AddAssign (VarIdentifier "a") (Type (Int 1))))
